inference/sources/fly/main.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/caarlos0/env/v6"
+	"github.com/dghubble/sling"
+	. "github.com/vlab-research/vlab/inference/inference-data"
+)
+
+func handle(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+type GetResponsesParams struct {
+	PageSize int    `url:"page_size"`
+	After    string `url:"after,omitempty"`
+}
+
+type GetResponsesResponse struct {
+	TotalItems int `json:"total_items"`
+	PageCount  int `json:"page_count"`
+	Items      []struct {
+		Parent_surveyid  string `json:"parent_surveyid"`
+		Parent_shortcode string `json:"parent_shortcode"`
+		Surveyid         string `json:"Surveyid"`
+		Flowid           string `json:"Flowid"`
```

Comment:
The json is capitalized here in some of these when it shouldn't be! -- your tests probably aren't testing these keys. If this connector uses the keys, they should be tested. If it doesn't, then it doesn't matter, but if they're in the type they should probably be written correctly :)